{{cookiecutter.folder_name}}/Makefile

```diff
@@ -15,6 +15,12 @@ main=main
 default=$(shell cat .metadata.json | grep 'default_format' | cut -d'"' -f4)
 outname=$(notdir $(shell pwd))
 
+ifeq ($(default), {{ cookiecutter.default_format }})
```

Comment:
All of the cookiecutter stuff makes it hard to test anything. If there is someway to set the default value of $default in the repo itself then I would be ok with this change, but having it set to the cookiecutter 'default_format' makes it impossible for me to develop.